src/shared/components/RichTable.tsx

```diff
@@ -196,51 +188,35 @@ const useRichTable = <D extends object = {}>({
     _setSearchQuery(query);
   };
 
-  const dataUrl = useMemo<string>(
+  const getWithParams = useCallback(
     () => {
-      let queryBuilder: { [key: string]: string | number } = {
+      let params: QueryParams = {
         time: Date.now(),
       };
       if (paginated) {
-        queryBuilder.page = page;
+        params.page = page;
       }
       if (searchQuery) {
-        queryBuilder.search = window.encodeURIComponent(searchQuery);
+        params.search = window.encodeURIComponent(searchQuery);
       }
       if (sortColumn !== null) {
-        queryBuilder.sort =
+        params.sort =
           (sortColumn[1] ? "" : "-") + window.encodeURIComponent(sortColumn[0]);
```

Comment:
Calls to encodeURIComponent are now unnecessary (as axios already serializes parameters itself).

Reply:
https://github.com/UWmathNEWS/slugline-web/pull/33/commits/9aa2bdd2f117930cdf0a9ce0dedbf1f7c6d186ac